cpu/efm32/include/periph_cpu.h

```diff
@@ -429,42 +429,73 @@ typedef struct {
  * @brief   UART device configuration.
  */
 #ifndef DOXYGEN
+/**
+ * @brief   Marker for unsupported UART modes
+ */
+#define UART_MODE_UNSUPPORTED 0xf0
+
 /**
  * @brief   Override parity values
  * @{
  */
 #define HAVE_UART_PARITY_T
+#if defined(_SILICON_LABS_32B_SERIES_0) || defined(_SILICON_LABS_32B_SERIES_1)
 typedef enum {
-   UART_PARITY_NONE = 0,
-   UART_PARITY_ODD = 1,
-   UART_PARITY_EVEN = 2,
-   UART_PARITY_MARK = 3,
-   UART_PARITY_SPACE = 4,
+    UART_PARITY_NONE = 0,
+    UART_PARITY_ODD = 1,
+    UART_PARITY_EVEN = 2,
+    UART_PARITY_MARK = 3,
+    UART_PARITY_SPACE = 4,
 } uart_parity_t;
+#else
+typedef enum {
+    UART_PARITY_NONE = 0,
+    UART_PARITY_EVEN = 2,
+    UART_PARITY_ODD = 3,
+    UART_PARITY_MARK = UART_MODE_UNSUPPORTED | 0,
+    UART_PARITY_SPACE = UART_MODE_UNSUPPORTED | 1,
```

Comment:
Why not use `UART_MODE_UNSUPPORTED` for both?

Reply:
`tests/periph_uart_mode` won't compile. The switch/case statement in `_get_mode()` complains about them being the same value ...